### PR TITLE
Renamed mmr items to mrr

### PR DESCRIPTION
**Pull request by matthewAURA**
I goofed.

### CODE REVIEW

**Comment by nallen01** [comment]
There's different case/spelling for the secret key environment variable in code vs readme

mrr_website/settings.py (line 8)
`MRR-WEBSITE_SECRET`

README.rst (lines 11 and 16)
`mrr_website_SECRET`

File: README.rst [hunk 1]
```diff
@@ -8,16 +8,16 @@ Mini Robot Rumble Website
 Quickstart
 ----------
 
-Before running shell commands, set the ``FLASK_APP``, ``FLASK_DEBUG`` and ``MMR_WEBSITE_SECRET``
+Before running shell commands, set the ``FLASK_APP``, ``FLASK_DEBUG`` and ``mrr_website_SECRET``
```
Comment (matthewAURA) [inline README.rst]: You can comment on files in the diff.

Comment (matthewAURA) [inline README.rst]: I'm also removing this in another PR, but I've fixed it up for now

Comment (nallen01) [inline README.rst]: I know, but you can't on the mobile website (as far as I can tell) and I was too lazy to change to desktop site.

**Pull request merged**